### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by zhijxu-MS**
string value "False" isn't equal to bool False.

if "False":
     call_function()  ## **this function will be called**.

### CODE REVIEW

**Comment by zhijxu-MS** [comment]
@liqunfu  there is a small bug in ort_trainer.py, could you help to take a look at it?

File: orttraining/orttraining/python/ort_trainer.py [hunk 1]
```diff
@@ -299,7 +299,7 @@ def convert_model_loss_fn_to_onnx(model, loss_fn, model_desc, device):
     other_export_options = {}
     # This option was added after 1.4 release.
     if LooseVersion(torch.__version__) > LooseVersion('1.4.0'):
-        other_export_options['enable_onnx_checker'] = 'False'
+        other_export_options['enable_onnx_checker'] = False
```
Comment (tianleiwu) [inline orttraining/orttraining/python/ort_trainer.py]: Good catch. 
As string 'False' is treated as True in if statement. Maybe change to True could align with existing behavior.

Comment (zhijxu-MS) [reply orttraining/orttraining/python/ort_trainer.py]: while i think bool False should be the expected value, right? @liqunfu

Comment (Tixxx) [reply orttraining/orttraining/python/ort_trainer.py]: Thanks for the fix. Yea, False is the intended value.

**Comment by edgchen1** [comment]
looks like this fix was already merged here: https://github.com/microsoft/onnxruntime/pull/3244

**Comment by SherlockNoMad** [comment]
Thank you for the fix. This fix has been merge in #3244. Closing the PR now.

**Pull request closed without merging**